.gitignore

```diff
@@ -3,3 +3,4 @@ bundle.js
 config.js
 jest.config.js
 coverage
+.env
```

Comment:
just a suggestion, since it doesn't like having the last line not being an empty line.